docs/canvas/canvas-elements.asciidoc

```diff
@@ -1,150 +1,115 @@
+[[add-canvas-elements]]
+=== Add elements
 
-Canvas _elements_ are the building blocks of your workpad. With elements, you can combine images, text, and visualizations to tell a story about your data.
-
-When you add elements to your workpad, you can:
-
-* <<connect-element-data,Connect the elements to your live data>>
-
-* <<configure-display-options,Choose how you want the element to appear on your workpad>>
-
-* <<configure-auto-refresh-interval,Change the data refresh interval>>
-
-* <<organize-element,Move and arrange elements on your workpad>>
-
-* <<element-save,Save your elements>>
+Create a story about your data by adding elements to your workpad that include images, text, charts, and more. You can create your own elements and connect them to your data sources, add saved objects that you created outside of Canvas, and add your own images.
 
 [float]
-[[add-canvas-element]]
-=== Add elements to your workpad
-
-Choose the elements to display on your workpad, then familiarize yourself with the element using the preconfigured demo data. By default, most elements use demo data until you change the data source. The demo data includes a small sample data set that you can use to experiment with your element.
+[[create-canvas-element]]
+==== Create an element
 
-To add a Canvas element:
+Choose the type of element you want to use, then connect it to your own data.
 
-. Click *Add element*.
-
-. In the *Elements* window, select the element you want to use.
+. Click *Add element*, then select the element you want to use.
 +
 [role="screenshot"]
 image::images/canvas-element-select.gif[Canvas elements]
 
-. Play around with the default settings and see what the element can do.
-
-To add a map:
+. To familiarize yourself with the element, use the preconfigured data demo data.
++
+By default, most of the elements you create use demo data until you change the data source. The demo data includes a small data set you can use to experiment with your element.
 
-. Click *Embed object*.
+. To connect the element to your data, select *Data*, then select one of the following data sources:
 
-. Select the map you want to add to the workpad.
-+
-[role="screenshot"]
-image::images/canvas-map-embed.gif[]
+* *{es} SQL* &mdash; Access your data in {es} using SQL syntax. For information about SQL syntax, refer to {ref}/sql-spec.html[SQL language].
 
-NOTE: Demo data is only supported on Canvas elements. Maps do not support demo data.
+* *{es} raw data* &mdash; Access your raw data in {es} without the use of aggregations. Use {es} raw data when you have low volume datasets, or to plot exact, non-aggregated values.
 
-Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right, then selecting *Delete*.
+* *Timelion* &mdash; Access your time series data using <<timelion,Timelion>> queries. To use Timelion queries, you can enter a query using the <<lucene-query,Lucene Query Syntax>>.
 
```

Comment:
Should Elasticsearch basic be included?  Also, should these items be listed in the same order as they are in the product?

Reply:
I'm going to document Elasticsearch basic in a separate PR after feature freeze. As of right now, these are the order that appears on the UI.